test/global-table.test.ts

```diff
@@ -30,5 +31,121 @@ test('Create global table with partition key and default properties', () => {
     Replicas: [{
       Region: newstack.region,
     }],
+    TableName: 'default_name',
   });
-});
\ No newline at end of file
+});
+
+test('Create global table with partition key and specified replica, tablename', () => {
+  const region = 'us-east-1';
+  const app = new App();
+  const newstack = new Stack(app, 'id', {
+    env: { region: region },
+  });
+  new GlobalTable(newstack, 'hello', {
+    partitionKey: {
+      name: 'id',
+      type: ddb.AttributeType.STRING,
+    },
+    replicas: [{
+      region: newstack.region,
+    },
```

Comment:
The customer should not have to supply their current region as one of the replicas. The current region should be the "built-in" copy of the data that is always created.